### PR TITLE
fix non clickable FPGA4fun links

### PR DESCRIPTION
**Pull request by marcmerlin**

### CODE REVIEW

**Comment by marcmerlin** [comment]
@tcal-x  tested the output and made sure links were fixed (i.e. clickable)

File: docs/source/crash-course/gateware.rst [hunk 1]
```diff
@@ -174,11 +174,11 @@ introduction to FPGAs we have found.
 Start off by reading through the material under FPGA introduction in the left
 hand menu (scroll way down). The most relevant pages are:
 
-* `What are FPGAs? <https://www.fpga4fun.com/FPGAinfo1.html>`
-* `How FPGAs work <https://www.fpga4fun.com/FPGAinfo2.html>`
-* `Internal RAM <https://www.fpga4fun.com/FPGAinfo3.html>`
-* `FPGA pins <https://www.fpga4fun.com/FPGAinfo4.html>`
-* `Clocks and global lines <https://www.fpga4fun.com/FPGAinfo5.html>`
+* `What are FPGAs?`: https://www.fpga4fun.com/FPGAinfo1.html
```
Comment (mithro) [inline docs/source/crash-course/gateware.rst:177]:
```suggestion
* `What are FPGAs? <https://www.fpga4fun.com/FPGAinfo1.html>`_
```

Comment (mithro) [reply docs/source/crash-course/gateware.rst:177]: These are missing the trailing '_'
```rst
`Link text <link URL>`_
```

**Comment by marcmerlin** [comment]
tested new rendering, works better even. thanks @mithro

**Review by alanvgreen — APPROVED** [review]

**Comment by alanvgreen** [comment]
Thanks!

**Pull request merged**